clouddrift/adapters/gdp1h.py

```diff
@@ -540,6 +540,16 @@ def to_raggedarray(
     a convenience method to emit a `xarray.Dataset` instance:
 
     >>> ds = ra.to_xarray()
+
+    To write the ragged-array dataset to a NetCDF file on disk, do
+
+    >>> ds.to_netcdf("gdp1h.nc", format="NETCDF4")
+
+    Alternatively, to write the ragged-array to a Parquet file, first create
```

Comment:
We're not consistent with the hyphens. I suggest we try ragged array instead of ragged-array.